src/frontend/src/optimizer/plan_node/logical_agg.rs

```diff
@@ -120,6 +123,80 @@ pub struct LogicalAgg {
     input: PlanRef,
 }
 
+impl LogicalAgg {
+    pub fn infer_internal_table_catalog(
+        &self,
+        plan_node: PlanRef,
```

Comment:
this parameter seems unnecessary?  and why implement it in logicalAgg instead of streamHashAgg and streamSimpleAgg?

Reply:
> this parameter seems unnecessary?
exactly. My fault.

> why implement it in logicalAgg instead of streamHashAgg and streamSimpleAgg?
because I think their implementations are same (maybe I miss something?